server/src/main/java/org/elasticsearch/index/mapper/DocumentParserContext.java

```diff
@@ -303,8 +298,13 @@ public boolean isCopyToField(String name) {
 
     /**
      * Add a new mapper dynamically created while parsing.
+     *
+     * @return returns <code>true</code> if the mapper could be created, <code>false</code> if the dynamic mapper has been ignored due to
+     * the field limit
+     * @throws IllegalArgumentException if the field limit has been exceeded and it's not possible to ignore the dynamic field according to
+     * the index settings
```

Comment:
is that because dynamic is set to true rather than to until_limit, or some other reason? Maybe we could clarify this comment?

Reply:
That's correct. I've tried to clarify it.